examples/basics/visuals/polygon_prototype.py

```diff
@@ -0,0 +1,49 @@
+from vispy import app
+import sys
+from vispy.scene import SceneCanvas
+from vispy.scene.visuals import Polygon, Ellipse, Rectangle
+from vispy.color import Color
+
+white = Color("#ecf0f1")
+gray = Color("#121212")
+red = Color("#e74c3c")
+blue = Color("#2980b9")
+orange = Color("#e88834")
```

Comment:
This is not really needed, but I felt like picking up colors from [flatuicolors](https://flatuicolors.com/) Just because they look so much nicer than the defaults